pkg/e2e/infra/utils.go

```diff
@@ -138,8 +138,8 @@ func getMachineSets(client runtimeclient.Client) ([]mapiv1beta1.MachineSet, erro
 	return machineSetList.Items, nil
 }
 
-// getMachineList returns a MachineList object
-func getMachineList(client runtimeclient.Client) (*mapiv1beta1.MachineList, error) {
+// getMachines returns the list of machines or an error
+func getMachines(client runtimeclient.Client) ([]mapiv1beta1.Machine, error) {
 	machineList := mapiv1beta1.MachineList{}
```

Comment:
why removing `getMachineList` instead of reusing it?

Reply:
I read this as the same thing, though instead of returning the `List` it returns a slice - the latter seems better IMO. And I'm sure I've requested similar changes in other PRs this week which do the same thing.

Reply:
based on the changes afterwards we probably want to drop this commit https://github.com/openshift/cluster-api-actuator-pkg/pull/49/commits/82456b63439e03c76fbc48b8a612fae3e24bbbb7?